features/org.jboss.tools.jst.js.feature/feature.properties

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+# Copyright (c) 2015 Red Hat, Inc. and others.
+# All rights reserved. This program and the accompanying materials 
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+# 
+# Contributors:
+#     JBoss by Red Hat - Initial implementation.
+##############################################################################
+
+# "featureName" property - name of the feature
+featureName=JBoss Tools JavaScript Tools
+
+# "providerName" property - name of the company that provides the feature
+providerName=JBoss by Red Hat
+
+# "updateSiteName" property - label for the update site
+updateSiteName=JBossTools Update Site
+
+# "description" property - description of the feature
+description=JavaScript Tools
+
+# "copyright" property - text of the "Feature Update Copyright"
+copyright=Copyright (c) 2015 Red Hat, Inc. and others.\n\
+All rights reserved. This program and the accompanying materials\n\
+are made available under the terms of the Eclipse Public License v1.0\n\
+which accompanies this distribution, and is available at\n\
+http\://www.eclipse.org/legal/epl-v10.html\n\nContributors\:\n\
+JBoss by Red Hat - Initial implementation.\n
+ ############### end of copyright property ###################################
+
+# START NON-TRANSLATABLE
+# "license" property - text of the "Feature Update License"
+# should be plain text version of license agreement pointed to be "licenseURL"
+license=Red Hat, Inc. licenses these features and plugins to you under \
```

Comment:
This is the old license. Please use new one from Foundation license feature. https://issues.jboss.org/browse/JBIDE-20206